src/Tabs/Tab.js

```diff
@@ -120,10 +120,18 @@ class Tab extends Component {
       };
       // If it's svg icon set color via props
       if (icon.type.muiName !== 'FontIcon') {
-        if (icon.props !== undefined && icon.props.style !== undefined && icon.props.style.color !== undefined) {
-          iconProps.color = icon.props.style.color;
-        } else {
-          iconProps.color = styles.root.color;
+        if (icon.props) {
```

Comment:
This all logic is getting too much complex. I think that the correct fix is the following:
```js
// If it's svg icon set color via props
if (icon.type.muiName === 'SvgIcon') {
  iconProps.color = icon.props.color || styles.root.color;
}
```